tests/functional/aws-node-sdk/test/multipleBackend/objectTagging/objectTagging.js

```diff
@@ -325,8 +326,8 @@ function testSuite() {
                 });
             });
 
-            it('should not return error on getting tags from object deleted ' +
-            'from AWS', done => {
+            it('should not return error on getting tags from object that has ' +
+            'had a delete marker put directly on AWS', done => {
```

Comment:
should we also test what we actually get back (beyond that no errror)?

Reply:
👍